templates.rst

```diff
@@ -604,6 +604,51 @@ to define the template to render::
 
     The ``#[Template()]`` attribute was introduced in Symfony 6.2.
 
+The ``AbstractController`` also provides the
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\AbstractController::renderBlock`
+and :method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\AbstractController::renderBlockView`
+methods::
+
+    // src/Controller/ProductController.php
+    namespace App\Controller;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+    use Symfony\Component\HttpFoundation\Response;
+
+    class ProductController extends AbstractController
+    {
+        // ...
+
+        public function price(): Response
+        {
+            // ...
+
+            // the `renderBlock()` method returns a `Response` object with the
+            // block contents
+            return $this->renderBlock('product/index.html.twig', 'price_block', [
+                // ...
+            ]);
+
+            // the `renderBlockView()` method only returns the contents created by the
+            // template block, so you can use those contents later in a `Response` object
+            $contents = $this->renderBlockView('product/index.html.twig', 'price_block', [
+                // ...
+            ]);
+
+            return new Response($contents);
+        }
+    }
+
+This might come handy when dealing with :ref:`templates inheritance <template_inheritance-layouts>`
```

Comment:
Is it template inheritance or template block ?
Difference between extends and block need to be explicit imho
Anyway, that was documented fast 🚀

Reply:
Reworded a bit to mention blocks (which are mentioned in the inheritance doc), does it seem better?